opensearch-observability/src/main/kotlin/org/opensearch/observability/index/ObservabilityIndex.kt

```diff
@@ -79,17 +80,15 @@ internal object ObservabilityIndex {
     fun initialize(client: Client, clusterService: ClusterService) {
         this.client = SecureIndexClient(client)
         this.clusterService = clusterService
+        this.mappingsUpdated = false
```

Comment:
i don't think this is needed? not very sure

Reply:
I was also not 100% sure, but I thought it wouldn't hurt.